README.md

```diff
@@ -1,79 +1,86 @@
+
 # BreezewayMod
+
 ## About
-![](breezewaymod_icon.png) 
 
-BreezewayMod is a fork of the Breezeway window decoration which aims to mimic the macOS interface. But with powerups.
-Did I mention that it comes without weird glitches on the rounded corners?
+<img src="breezewaymod_icon.png" style="zoom: 25%;" /> 
 
-## Screenshots
-![](screenshot.png)
+**BreezewayMod** is a fork of the [Breezeway](https://github.com/richard-kung/breezeway) window decoration, which aims to mimic the *macOS style*, but with **power ups**.
 
-Looks good with light and dark color schemes (shown here with color schemes "Arc" for light windows and "Arc-Dark" for dark windows)
+> **Note**: it comes without weird glitches on the rounded corners.
```

Comment:
This kind of loses the tongue-in-cheek tone of the current text. If we're pointing out that it doesn't have the glitches in a more serious tone, it'd be good to also link a documentation on what that glitch actually is.